AdminUi/src/AdminUi/ClientApp/src/app/components/client/client-list/client-list.component.ts

```diff
@@ -74,17 +79,67 @@ export class ClientListComponent {
             });
     }
 
-    pageChangeEvent(event: PageEvent) {
+    pageChangeEvent(event: PageEvent): void {
         this.pageIndex = event.pageIndex;
         this.pageSize = event.pageSize;
         this.getPagedData();
     }
 
-    dateConvert(date: any) {
+    dateConvert(date: any): string {
         return new Date(date).toLocaleDateString();
     }
 
-    addClient() {
+    addClient(): void {
         this.router.navigate([`/clients/create`]);
     }
+
+    deleteClient(): void {
+        this.loading = true;
+        let observableBatch: Observable<any>[] = [];
+        this.selection.selected.forEach(item => {
+            observableBatch.push(this.clientService.deleteClient(item.clientId));
+        });
+
+        forkJoin(observableBatch)
+            .subscribe({
+                next: (data: any) => {
+                    this.getPagedData();
+                    this.snackBar.open('Successfully deleted clients.', 'Dismiss', {
```

Comment:
Shouldn't it be singular?
```suggestion
                    this.snackBar.open('Successfully deleted client.', 'Dismiss', {
```

Reply:
I can check whether we attempted to delete multiple or just a single client and display the message accordingly.

Reply:
This is of course not necessary due to the other comment

Reply:
Didn't see your comment before I posted mine. Maybe your suggestion is a good idea. Or even better: Write "Successfully deleted X clents".

Reply:
Will do! 👍🏼